docs/naivebayes.md

```diff
@@ -0,0 +1,108 @@
+Naive Bayes Classifiers
+=======================
+
+The Naive Bayes classifiers is a simple probabilistic classification model based on Bayes Theorem. It is a simple, yet powerful classification model. Since Naive Bayes classifiers simply classifies sets of data by which class has the highest likelihood, Naive Bayes classifiers can use any dsitribution or model that outputs a probability.
```

Comment:
It should be made explicit that we mean the classifier can use any distribution or model with a `log_probability` method, even though theoretically yes it's any model with a probability.